CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -131,18 +131,24 @@ Please find more information in the [Import PDFs and images guide](https://devel
 
 ### Camera import error handling TODO
 
-## Review screen TODO
+## Review screen
 
 <iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F1985HMF83siAXmysSn3dC6%2FiOS-Gini-Capture-SDK-3.0.0-UI-Customisation%3Fnode-id%3D261%253A8256%26t%3DcRAvcUKVlwGtGpuh-1" allowfullscreen></iframe>
 
+You can show a custom loading indicator with custom animation support on the process button.
+Your custom loading indicator should implement `OnButtonLoadingIndicatorAdapter` interface and be passed  to `GiniConfiguration.shared.onButtonLoadingIndicator`.
+onButtonLoadingIndicator: OnButtonLoadingIndicatorAdapter
+
+The example implementation is available [here](https://github.com/gini/gini-mobile-ios/blob/7e18d704c15c8929a97c88226724836e3b477cf1/BankSDK/GiniBankSDKExample/GiniBankSDKExample/CustomLoadingIndicator.swift#L36).
```

Comment:
[Request] The link should ideally point to a tag so that the documentation version is in sync with code examples. 

Until we can automate updating the link we will should do it manually. Even if we forget it, searching for the version name we can find it later.

```suggestion
The example implementation is available [here](https://github.com/gini/gini-mobile-ios/blob/GiniCaptureSDK%3B3.0.0-beta01/BankSDK/GiniBankSDKExample/GiniBankSDKExample/CustomLoadingIndicator.swift#L36).
```